### DISCUSSION TITLE
March 2021: backups with Plakar · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#128] [Articles]

**Comment by erdnaxeli** [comment]
Hi,

Great project!

I think there is a typo, because both example commands for inode diff and diff are the same :)

**Comment by poolpOrg** [comment]
thanks @erdnaxeli, I'm either too tired or have bad sight, I can't seem to find the pasto :-/

**Comment by erdnaxeli** [comment]
It's line 614 (on the source file). But reading it again maybe it's OK, the two diff commands do not have the same arguments.

**Comment by Glandos** [comment]
Hi,

I see that plakar uses all good ideas from borgbackup and restic, that you should know about ;)

So my little question is: how it is different from other solutions?

The network part seems a bit new, but it was possible using SSH. There is also the namespace feature, but I'm using borg with different repositories (or store in plakar jargon) to have the same effects, with the benefit of different access rights it.

**Comment by poolpOrg** [comment]
As of today, it isn't much different from others, besides the way you interact with stores, I wouldn't switch from borg or restic if I were using them to plakar if that's your question.
